mne/channels/montage.py

```diff
@@ -606,7 +606,7 @@ def _get_montage_in_head(montage):
 
 
 @fill_doc
-def _set_montage(info, montage, match_case=True):
+def _set_montage(info, montage, match_case=True, on_missing=True):
```

Comment:
This is not a valid value. Somewhere below you should do:
```
_check_option('on_missing', on_missing, ('raise', 'ignore'))
```

Reply:
this is not fixed